packages/pieces/postgres/src/lib/actions/run-query.ts

```diff
@@ -0,0 +1,95 @@
+import { createAction, Property } from "@activepieces/pieces-framework";
+import pg from 'pg';
+
+export const runQuery = createAction({
+    name: 'run-query',
+    displayName: "Run Query",
+    description: "Run Query",
+    props: {
+        authentication: Property.CustomAuth({
+            displayName: "Authentication",
+            props: {
+                host: Property.ShortText({
+                    displayName: 'Host',
+                    required: true,
+                }),
+                port: Property.ShortText({
+                    displayName: 'Port',
+                    required: true,
+                }),
+                user: Property.ShortText({
+                    displayName: 'User',
+                    required: true,
+                }),
+                password: Property.SecretText({
+                    displayName: 'Password',
+                    required: true,
+                }),
+                database: Property.ShortText({
+                    displayName: 'Database',
+                    required: true,
+                })
+            },
+            required: true
+        }),
+        query: Property.ShortText({
+            displayName: 'Query',
+            required: true,
+        }),
+        rejectUnauthorized: Property.Checkbox({
+            displayName: 'Reject Unauthorized',
+            required: true,
+            defaultValue: true
+        }),
+        keepAlive: Property.Checkbox({
+            displayName: 'Reject Unauthorized',
+            required: false,
+        }),
+        query_timeout: Property.Number({
```

Comment:
Please add description for the props.

- `host`: A string indicating the hostname of the PostgreSQL server to connect to. If omitted, the default is `localhost`.

- `user`: A string indicating the user to authenticate as when connecting to the PostgreSQL server. This parameter is required.

- `password`: A string indicating the password to use for authentication. If omitted, the client will attempt to connect to the server using other authentication methods (such as an authentication token or the user's operating system credentials).

- `database`: A string indicating the name of the database to connect to. This parameter is required.

- `query_timeout`: An integer indicating the maximum number of milliseconds to wait for a query to complete before timing out. If omitted, there is no query timeout.

- `application_name`: A string indicating the name of the client application connecting to the server. This can be useful for identifying which client applications are using the server's resources. If omitted, the default is to use the name of the calling Node.js process.

- `connectionTimeoutMillis`: An integer indicating the maximum number of milliseconds to wait for a connection to be established before timing out. If omitted, the default is to wait indefinitely.